### PR TITLE
TEN-1455 / 24.10 / Adding 2 test cases to test system dataset in the UI

### PR DESCRIPTION
**Pull request by ericbsd**

### CODE REVIEW

**Comment by ericbsd** [comment]
image

File: test_cases/webui/system_settings/advanced/test_advenced_storage.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    def test_changing_storage_system_dataset_pool_is_kept_after_reboot(self):
+        """
+        Summary: This test verifies the admin is able to change the storage system dataset pool
+         and the pool is kept after reboot.
+
+        Test Steps:
+        1. Navigate to System Settings > Advanced.
+        2. One the Storage card click configure.
+        3. Change the storage system dataset pool to new-tank.
+        3. Verify the storage system dataset pool is changed on the UI and on the API.
```
Comment (markbainix) [inline test_cases/webui/system_settings/advanced/test_advenced_storage.py:41]: numbering off.

File: test_cases/webui/system_settings/advanced/test_advenced_storage.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    @allure.tag('Update')
+    @allure.story('Changing Storage System Dataset Pool Is Kept After Reboot')
+    def test_changing_storage_system_dataset_pool_is_kept_after_reboot(self):
+        """
+        Summary: This test verifies the admin is able to change the storage system dataset pool
+         and the pool is kept after reboot.
+
+        Test Steps:
+        1. Navigate to System Settings > Advanced.
+        2. One the Storage card click configure.
```
Comment (markbainix) [inline test_cases/webui/system_settings/advanced/test_advenced_storage.py:39]: is this supposed to be "On"?

File: test_cases/webui/system_settings/advanced/test_advenced_storage.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        3. Change the storage system dataset pool to new-tank.
+        3. Verify the storage system dataset pool is changed on the UI and on the API.
+        5. Reboot and log back in.
+        6. Verify the storage system dataset pool is changed on the UI and on the API.
+        """
+        NAV.navigate_to_system_settings_advanced()
+        COM.is_card_visible('Storage')
+        COM.click_button('storage-configure')
+        COM.is_dialog_visible('Warming')
```
Comment (markbainix) [inline test_cases/webui/system_settings/advanced/test_advenced_storage.py:48]: should this be "assert"ed to ensure the dialog is present

**Review by markbainix — CHANGES_REQUESTED** [review]

**Review by markbainix — APPROVED** [review]

**Review by markbainix — APPROVED** [review]

**Pull request merged**